modules/eks/ray-orchestrator/README.md

```diff
@@ -0,0 +1,121 @@
+# Ray Orchestrator
+
+## Description
+
+This module orcehstrates submission of a training job to the Ray Cluster using AWS Step Functions.
+
+## Inputs/Outputs
+
+### Input Parameters
+
+#### Required
+
+- `namespace` - Kubernetes namespace name
+- `eks_cluster_admin_role_arn`- ARN of EKS admin role to authenticate kubectl
+- `eks_handler_role_arn`- ARN of EKS admin role to authenticate kubectl
+- `eks_cluster_name` - Name of the EKS cluster to deploy to
+- `eks_cluster_endpoint` - EKS cluster endpoint
+- `eks_oidc_arn` - ARN of EKS OIDC provider for IAM roles
+- `eks_cert_auth_data` - Auth certificate
+
+#### Optional
+
+- `step_function_timeout` - Step function timeout in minutes
+- `data_bucket_name` - Name of the bucket to grant service account permissions to
+- `tags` - List of additional tags to apply to all resources
```

Comment:
I think tags is a map of string to string?

Reply:
Thanks, updated.